<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin::Internal::MainWindow</class>
 <widget class="QMainWindow" name="MainWin::Internal::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="margin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Patient Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="patientName"/>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QGroupBox" name="patientInformations">
      <property name="title">
       <string>Patient informations</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="dobLabel">
           <property name="text">
            <string>DOB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dobDateEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Weight (kg)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="patientWeight">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Size (cm)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="patientSize">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Gender</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sexCombo">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>F</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>H</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Creatinin clearance (ml/min)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="patientClCr">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>170</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Creatinin (mg/l)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="patientCreatinin">
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>List of medications allergies</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="listOfAllergies"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="DrugsWidget::DrugsCentralWidget" name="m_CentralWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="morePatientInfoButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrugsWidget::DrugsCentralWidget</class>
   <extends>QWidget</extends>
   <header location="global">drugsplugin/drugswidget/mfDrugsCentralWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>morePatientInfoButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>patientInformations</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
